README.md

```diff
@@ -1,3 +1,140 @@
+- **Spring Security**: Security is a top priority. We've implemented Spring Security to protect your data and ensure only authorized access.
+- **JWT (JSON Web Tokens)**: Fortify our application with JWT-based authentication and authorization, as if you're locking it down with a secure shield.
+- **Spring Data JPA**: With Spring Data JPA, managing our book data in the database is a breeze.
+- **MapStruct**: MapStruct simplifies the transformation between different data models.
+- **Liquibase**: Effortlessly manage database schema changes with Liquibase, making it seem like a walk in the park.
+- **Swagger**: Explore and interact with our API easily through the Swagger documentation.
+- **Docker**: Containerization for easy deployment and scalability.
+- **Postman**: I've provided a collection of Postman requests to help you test and use the API effectively.
+
+## Controller Functionalities
+
+### Our API consists of several controllers, each serving a specific purpose:
+
+- **AuthenticationController**: Handles user authentication and registration.
+
+- **BookController**: Handles book-related operations, including creation, retrieval, update, and deletion of books.
+
+- **CategoryController**: Manages categories, enabling admins to create, update, delete categories, and retrieve books by category.
+
+- **ShoppingCartController**: Manages shopping cart information, allowing users to create, retrieve, update, and delete cart items.
+
+- **OrderController**: Handles order management, including creating orders, updating order status, and retrieving order history.
+
+# Installation and Usage
+
+You can interact with the Book Store API in two ways:
+
+1. **Using Swagger (No Installation Required)**
+
+   To test the API without any installation, you can utilize Swagger, which provides a user-friendly interface for interacting with your API.
+
+    - Open your web browser and navigate to [Swagger UI](http://ec2-13-51-233-107.eu-north-1.compute.amazonaws.com/api/swagger-ui/index.html#/).
+    - Explore and test the various API endpoints directly through the Swagger interface.
+
+**Authentication for Swagger UI:**
+
+- **Admin Access**: If you want to access admin-specific functionalities, use these credentials:
+    - **Username**: admin@a.ua
+    - **Password**: qwerty123
+
+2. **Local Installation**
+
+   If you prefer to run the Book Store API on your local machine, follow these steps:
+
+   ### Prerequisites
+
+   Before you begin, ensure you have the following prerequisites installed on your system:
+
+    - [Java 17](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html)
+    - [MySQL database](https://www.mysql.com/downloads/)
+    - [Docker](https://docs.docker.com/get-docker/)
+
+   ### Installation
+
+   Follow these steps to set up and run the Book Store API on your local machine:
```

Comment:
add alternative way to run app

Reply:
I thought that AWS and locally enough, but tbh i have no idea what another way to add

Reply:
I mean without docker